WonSeungHyun/app.js

```diff
@@ -37,3 +35,118 @@ const start = async () => {
 };
 
 start();
+
+app.post("/users", async (req, res, next) => {
+  const { name, email, profile_image } = req.body;
+
+  await myDataSource.query(
+    `
+    INSERT INTO users(
+    name,
+    email,
+    profile_image
+    
+  )VALUES (?, ?, ?)`,
+    [name, email, profile_image]
+  );
```

Comment:
- SQL 문법도 가독성 좋게 만들어주는 것이 중요합니다. 회사마다 컨벤션이 다르겠지만, 위코드에서는 아래와 같은 SQL 문법의 컨벤션을 정하고 있습니다.
- 지금은 위코드에서 사용하고 있는 컨벤션을 지켜서 문법을 작성해주세요. 하지만 이는 회사마다 다르기 때문에, 정답이 아님을 반드시 인지해주세요.
- SQL 포맷팅은 다음 처럼 통일하면 좋을거 같습니다.
```suggestion
  // Example
await database.query(`
      INSERT INTO users (
          name,
          email,
          profile_image,
          password
      ) VALUES (?, ?, ?, ?)`,
      [ name, email, profile_image, password ]
);
```